packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.mdx

```diff
@@ -15,10 +14,17 @@ import * as textMappingStories from '../../../mapping/text/tests/mapping-text.st
 <Title of={tableColumnEnumTextStories} />
 <Description of={tableColumnEnumTextStories} />
 
-<Canvas of={tableColumnEnumTextStories.enumTextColumn} />
-<ArgTypes of={tableColumnEnumTextStories.enumTextColumn} />
+<DocsStory of={tableColumnEnumTextStories.enumTextColumn} expanded={false} />
+<Controls of={tableColumnEnumTextStories.enumTextColumn} />
 
-### Text Mapping
+# Usage Docs
+
+## Blazor Usage
+
+When setting a child mapping elment's `Key` value to a string, you must wrap it in `@()` so that the compiler treats it as a string, e.g.
```

Comment:
```suggestion
When setting a child mapping element's `Key` value to a string, you must wrap it in `@()` so that the compiler treats it as a string, e.g.
```